### DISCUSSION TITLE
CSLA Discord server

### DISCUSSION THREAD

**Discussion by rockfordlhotka** [MarimerLLC/csla#2018] [General]
Hello everyone!

Discord is the new hot thing for community interaction. 

We currently have this forum and a [gitter channel](https://gitter.im/MarimerLLC/csla) for collaboration. Though the gitter channel is rarely used.

I'm wondering if I should have a Discord channel for CSLA. 

One motivation for this is that CSLA 6 work is going to get underway in the near future, and quite a few people are already very involved in discussions here in the repo around planning and designing some of the features for the new version. Discord could offer us a way to have meetings (audio/video) for such planning. The big challenge is timezones, but maybe we can sort that out?

Thoughts?

**Comment by rockfordlhotka** [comment]
Twitter thread: https://twitter.com/cslanet/status/1346546190981148678?s=20

**Comment by rockfordlhotka** [comment]
To thunderous silence, but because I want to keep up with the cool kids, I did create a Discord server for CSLA.

https://discord.gg/9ahKjb7ccf

**Reply by TheCakeMonster** [reply]
Oh, you're just _so_ trendy!

Next you'll be encouraging us to move off WebForms onto a modern web development framework like MVC, or something ;-)

Yes, it's probably a good plan, I think. The Gitter client for Android was, erm, a bit of a disappointment; it worked fine on my tablet, but just didn't size correctly on my mobile. I was surprised that people could get on with it, but maybe the iOS client was better.

And it's always good to have another login ... _gulp_
